powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/AuditingServiceBehavior.java

```diff
@@ -195,6 +195,7 @@ public void logSignatureAuditRecord(ActivationRecordDto activation, SignatureDat
                 .param("activationVersion", version)
                 .param("note", note)
                 .param("timestamp", currentTimestamp)
+                .param("type", AuditType.SIGNATURE.getCode())
```

Comment:
Let's make it consistent and place it at the very beginning.
```suggestion
                .type(AuditType.SIGNATURE.getCode())
```